tests/test_behavior.py

```diff
@@ -433,6 +434,13 @@ def test_mixed_english_chinese(self):
         )
 
 
+class FillDef(Base):
+    def test_trailing_new_line_removed(self):
+        note = {"English": ""}
+        assert fill_def("浴缸", note, "en")
+        self.assertEqual(note["English"], " \tbathtub")
```

Comment:
Wait, was this a broken case in the first place? bathtub doesn't end with `r` or `s` ?

Reply:
Lol thanks good catch, this is actually testing the bug it's just that I swapped the letter in the commit message, it should actually be `r`  and `b` which were being chopped off.

Reply:
Okay, I updated the commit message and the title of the PR.